README.md

```diff
@@ -126,3 +126,74 @@ To do that, simply run:
 ```bash
 ./cna-installer cluster create
 ```
+
+## Creating a Cluster from scratch using docker
+
+Few notes how to use the `cna-installer` from scratch using Docker.
+
+Run Docker container:
+
+```bash
+docker run --rm -it -e USER="$USER" -e ARM_CLIENT_ID="$ARM_CLIENT_ID" -e ARM_CLIENT_SECRET="$ARM_CLIENT_SECRET" -e ARM_SUBSCRIPTION_ID="$ARM_SUBSCRIPTION_ID" -e ARM_TENANT_ID="$ARM_TENANT_ID" -e SSH_AUTH_SOCK=$SSH_AUTH_SOCK -v $HOME/.ssh:/root/.ssh:ro -v $SSH_AUTH_SOCK:$SSH_AUTH_SOCK ubuntu
+```
+
+Execute following command in Docker container...
+
+Install terraform + git and other handy tools:
+
+```bash
+apt update
+apt install -y curl git golang jq unzip
+```
+
+Install Terraform:
+
+```bash
+TERRAFORM_LATEST_VERSION=$(curl -s https://checkpoint-api.hashicorp.com/v1/check/terraform | jq -r -M '.current_version')
```

Comment:
please note that the terraform build target is pinned to version 0.12.9, and it is fetched to the system via 'make build'

Reply:
Fixed....